crates/log-server/src/loglet_worker.rs

```diff
@@ -199,11 +199,16 @@ impl<S: LogStore> LogletWorker<S> {
                 }
                 Some(_) = in_flight_stores.next() => {}
                 // The in-flight seal (if any)
-                Some(Ok(_)) = &mut in_flight_seal => {
+                Some(result) = &mut in_flight_seal => {
                     sealing_in_progress = false;
-                    self.loglet_state.get_local_tail_watch().notify_seal();
-                    debug!(loglet_id = %self.loglet_id, "Loglet is now sealed on this log-server node");
                     in_flight_seal.set(None.into());
+
+                    if let Err(err) = result {
+                        trace!(loglet_id = %self.loglet_id, %err, "Seal operation has been aborted");
```

Comment:
We cannot really allow this to happen since we will be already reporting that the loglet is SEALING. The only error that can happen here is if the rocksdb writer crashed. If this happens, then we might as well panic/shutdown.

Reply:
Ah, I see. In `process_store` we send `SEALING` back, right? Will update this part then to panic if it should happen.  Right now, the TaskCenter should stop these tasks before stopping RocksDB, right?